app/policies/manual_policy.rb

```diff
@@ -0,0 +1,28 @@
+class ManualPolicy < ApplicationPolicy
+  def index?
+    true
+  end
+
+  alias_method :new?, :index?
+  alias_method :create?, :index?
+  alias_method :edit?, :index?
+  alias_method :update?, :index?
+  alias_method :show?, :index?
+
+  class Scope
```

Comment:
I know that the commit message for adding this `Scope` says no tests were included because it's a refactoring already covered by a feature spec, and I mostly agree with the sentiment.  But... this scope is fairly important and to have the documentation for how it works hidden in a feature spec seems a little opaque so I'm somewhat conflicted.  Anyone else have thoughts?